projects/microphysics/argo/argo.yaml

```diff
@@ -0,0 +1,376 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: prognostic-microphysics-
+  labels:
+    project: microphysics-emulation
+    job_type: prognostic-evaluation
+spec:
+  entrypoint: prognostic-run
+  arguments:
+    parameters:
```

Comment:
One missing feature I would like is a way to group multiple runs (but different `workflow.name`) under the same W&B "run group".  It seems like it's a prominent feature in the UI and it was really helpful for comparison of my recent experiments.

Perhaps just an option to override the default of the job name, which works well for single experiments.

Reply:
I added `wandb-group` and `wandb-tags` parameters.